tools/getPackageJson.js

```diff
@@ -0,0 +1,15 @@
+import fs from 'fs';
+import path from 'path';
+
+const getPackageJson = function(...args) {
+  const packageJSON = JSON.parse(fs.readFileSync(path.join(__dirname, '../package.json')));
+  if (!args.length) {
+    return packageJSON;
+  }
+  return args.reduce((out, key) => {
+    out[key] = packageJSON[key];
+    return out;
+  }, {});
+};
+
+module.exports = getPackageJson;
```

Comment:
You can use `export default getPackageJson`

Reply:
@ayusharma be careful whether that works on Node6

Reply:
Ohh.  Maybe it is fine then... 👍

Reply:
We are not running functional testing on Node6 anymore, because of lack of ES6 support it a huge pain. Node 6 will be only supported for us this year on build pipeline.